scripts/superdesk-monitoring/monitoring.js

```diff
@@ -423,18 +422,44 @@
                     }
                 });
 
-                scope.$on('item:highlight', queryItems);
+                scope.$on('item:highlight', scheduleQuery);
+                scope.$on('content:update', scheduleIfShouldUpdate);
+
+                if (scope.group.type !== 'stage') {
+                    scope.$on('ingest:update', scheduleQuery);
+                }
 
                 function scheduleIfShouldUpdate(event, data) {
-                    if (event && data && cards.shouldUpdate(scope.group, data)) {
+                    if (data.from_stage && data.from_stage === scope.group._id) {
+                        // item was moved from current stage
+                        extendItem(data.item, {
+                            gone: true,
+                            _etag: data.from_stage // this must change to make it re-render
+                        });
+                    } else if (data.to_stage && data.to_stage === scope.group._id) {
+                        // new item in current stage
+                        scheduleQuery();
+                    } else if (data && cards.shouldUpdate(scope.group, data)) {
                         scheduleQuery();
                     }
                 }
 
-                scope.$on('item:fetch', scheduleIfShouldUpdate);
-                scope.$on('content:update', scheduleIfShouldUpdate);
+                function extendItem(itemId, updates) {
+                    scope.$apply(function() {
+                        scope.items._items = scope.items._items.map(function(item) {
```

Comment:
Can you please check if the `extendItem` works for the `published` items i.e. there will be multiple versions of the same item in the published collection?